pkg/cli/globalflags/globalflags.go

```diff
@@ -8,6 +8,8 @@ import (
 type GlobalOptions struct {
 	BackplaneURL string
 	ProxyURL     string
+	Manager      bool
+	Service      bool
 }
 
 func AddGlobalFlags(cmd *cobra.Command, opts *GlobalOptions) {
```

Comment:
This globalflags makes me a bit confusing. Is this something should be shared across all the subcommands?
But looks there is only one globalflag for backplane cli which is `verbosity` https://github.com/openshift/backplane-cli/blob/b84a15bb5dfe6ae3fa52b9a337fbdb40fdd26fa6/cmd/ocm-backplane/root.go#L63

Reply:
These flags aren't meant to be shared across all subcommands. For example, `ocm backplane mangedjob create ...`, `ocm backplane scripts ...`, etc are meant to be confined to the cluster the user is logged into.

Reply:
It sounds something we need to refactor to group the flags, and leave the global option real global

Reply:
Thanks for merging, didn't get time to get back to this, we'll refactor global flags into flag groups soon.